CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+ - Invalid user dictionaries are handled more robustly [[#28]][28]
+
+[28]: https://github.com/envato/zxcvbn-ruby/pull/28
```

Comment:
I think GitHub automatically adds a link to PRs whenever you add a # before its number.

```suggestion
 - Invalid user dictionaries are handled more robustly [#28]
```